stress/shell/src/main/java/alluxio/stress/cli/MasterBatchTask.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.stress.cli;
+
+import alluxio.stress.Summary;
+import alluxio.stress.master.MasterBatchTaskParameters;
+import alluxio.util.JsonSerializable;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.ParametersDelegate;
+import com.google.common.collect.ImmutableMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Multiple tasks runner for Master StressBench.
+ */
+public class MasterBatchTask extends BatchTask {
+  private static final Logger LOG = LoggerFactory.getLogger(MasterBatchTask.class);
+  private static final int TARGETTHROUGHPUT = 1000000;
+  private static final String WARMUP = "0s";
+  private static final String DURATION = "21600s";
+  private static final String CLUSTER = "--cluster";
+
+  private String mOperation;
+
+  @ParametersDelegate
+  private MasterBatchTaskParameters mParameter = new MasterBatchTaskParameters();
+
+  @Override
+  public void run(String[] args) {
+    mOperation = args[0];
+    String[] input = Arrays.copyOfRange(args, 1, args.length);
+    JCommander jc = new JCommander(this);
+    jc.setProgramName(this.getClass().getSimpleName());
+    try {
+      jc.parse(input);
+    } catch (Exception e) {
+      LOG.error("Failed to parse command: ", e);
+      jc.usage();
+      throw e;
+    }
+
+    List<String[]> command = getCommand();
+    List<Summary> results = new ArrayList<>();
+    StressMasterBench bench = new StressMasterBench();
+    for (String[] arg : command) {
+      //error on individual task will not end the whole batch task
+      try {
+        String jsonResult = bench.run(arg);
+        Summary result = (Summary) JsonSerializable.fromJson(jsonResult);
+        results.add(result);
+      } catch (Exception e) {
```

Comment:
There are probably two kinds of exceptions:
1. You should stop here because there's no chance to continue
2. You can log this exception and move on
Are you going to handle these separately?

Reply:
Currently, I want to log the exception and let it move on. I am considering which kind of error should stop the whole task. If there are conditions which should result in stopping the batch task, I will add them here.